test/e2e/scripts/utils/local_system.py

```diff
@@ -76,3 +75,16 @@ def run(
         timeout=timeout_sec,
         check=True
     )
+
+@allure.step('Get pid by process name')
+def get_pid_by_process_name(name):
+    if get_platform() == "Windows":
+        name = "{}.exe".format(name)
```

Comment:
```suggestion
        name = "Status..exe".format(name)
```
It is not `nim_status_client` on Windows, it is `Status.exe`

![image](https://github.com/user-attachments/assets/bc9d3e49-a741-4068-87be-1da88ded4905)